src/tracks/InstrumentTrack.cpp

```diff
@@ -124,6 +124,7 @@ InstrumentTrack::InstrumentTrack( TrackContainer* tc ) :
 	connect( &m_baseNoteModel, SIGNAL( dataChanged() ), this, SLOT( updateBaseNote() ) );
 	connect( &m_pitchModel, SIGNAL( dataChanged() ), this, SLOT( updatePitch() ) );
 	connect( &m_pitchRangeModel, SIGNAL( dataChanged() ), this, SLOT( updatePitchRange() ) );
+	connect( &m_mutedModel, SIGNAL( dataChanged() ), this, SLOT( muteHasChanged() ) );
```

Comment:
These are the signal listeners we have today on `InstrumentTrackView`: https://github.com/LMMS/lmms/blob/master/src/tracks/InstrumentTrack.cpp#L845-849

To be frank, I feel fine with `muteHasChanged()`. Maybe even `muteChanged()` would be fine, this makes explicit that it is an event that is triggered when some model was muted.